### ISSUE TITLE
Add maykin_2fa

### ISSUE THREAD

**Issue by stevenbal** [open-zaak/open-zaak#1637]
I thought 2FA was already part of Open Zaak and Open Notificaties, but it turns out that was never the case. I think it would be good idea to add that, just like it was added in Objects/Objectypes API

https://maykin-2fa.readthedocs.io/en/stable/

**Comment by joeribekker** [comment]
Since, this is up... I would really like to add the same construction we did in Open Formulieren: based on the auth backend, use 2fa or not. And ofcourse, add 2fa to OZ and ON :)

For the one pickinf this up, please updated this ticket to reflect what needs to be done in all the projects :)

**Comment by joeridiederen** [comment]
Hi @joeribekker; weet je wanneer deze optie in Open Notificaties gaat landen? Ik zie dat dit ticket zowel OZ als ON beschrijft, maar op de ON-backlog vind ik 'm niet terug (https://github.com/open-zaak/open-notificaties/milestones). Alvast dank!

**Comment by joeribekker** [comment]
@joeridiederen klopt, we merkten dat we steeds tickets kopieerden van component A naar B dus we proberen iets meer te centraliseren maar daardoor zie je het niet direct op de backlog van het component.

**Comment by joeribekker** [comment]
Het is overigens wel gedaan: https://github.com/open-zaak/open-notificaties/pull/151
